homeassistant/const.py

```diff
@@ -758,7 +758,9 @@ class UnitOfVolumeFlowRate(StrEnum):
     """Volume flow rate units."""
 
     CUBIC_METERS_PER_HOUR = "m³/h"
+    CUBIC_METERS_PER_SECOND = "m³/s"
     CUBIC_FEET_PER_MINUTE = "ft³/m"
+    LITERS_PER_SECOND = "L/s"
```

Comment:
Could you also add "L/m"?
https://github.com/home-assistant/core/blob/0c66346fb089acee3cd6bb3fed761679fc371e9d/homeassistant/components/opentherm_gw/const.py#L298

Reply:
Done. But are we sure about the `/m` for `per minutes`? It sounds like `per meter`. I was expecting `/min`

Reply:
Ah indeed! I'm not sure what is best here as CUBIC_FEET_PER_MINUTE uses `m`
Cc @emontnemery

Reply:
Since we are unsure here, maybe it is best to simply ignore it for now.

Reply:
Well, let’s wait a for an other point of view on this.

Reply:
Based on a discussion on Discord with @emontnemery: we should add as `L/min`
But there is a caveat: maybe an architecture discussion is needed to add these three new enum values.

Separately, we should open a separate "breaking-change" PR to adjust `ft³/m` => `ft³/min` (and add to EQUIVALENT_UNITS in sensor/recorder.py)
https://github.com/home-assistant/core/blob/0c66346fb089acee3cd6bb3fed761679fc371e9d/homeassistant/components/sensor/recorder.py#L55-L61

Reply:
Perfect!

Reply:
We normally don't add common units etc until at least a handful of integrations use them. I think we should wait until more integrations use these units before we add them. Thanks!

Reply:
@MartinHjelmare that does not really make as much sense since it's now also used for unit conversions in frontend for display purposes.

Reply:
I think it still makes sense. We don't want to add all possible units just because they exist. Either there should be an apparent need in many integrations or we decide to add the unit after an architecture discussion.

Reply:
I think we should do it in order:
- first the device class
- second thr unit conversion for existing units
- third considrr adding new units